starter-code/apple-pie-starter-code/css/main.css

```diff
@@ -0,0 +1,46 @@
+body{
+  font-family: Arial, Helvetica, sans-serif;
+}
+h1{
+  color: white;
+  background-image: url('/Users/Kristen/Desktop/lab-css-recipes-clone/starter-code/apple-pie-starter-code/images/apple-pie.jpg');
```

Comment:
to link your image to the folder its located in your would set the `url('../images/apple-pie.jpg')`